### PR TITLE
 Don't trigger the zoom on legends area

### PR DESCRIPTION
**Pull request by ikkala**
Fixes chartjs#256

### CODE REVIEW

**Review by dfriday — APPROVED** [review]

**Comment by zloveless** [comment]
Any movement on merging this? Is this plugin actively being developed or maintained?

**Comment by zloveless** [comment]
@igorlukanin @etimberg @kurkle @nnnick @simonbrunel This plugin doesn't seem to be maintained. Is there anyone from the chart.js project that would be willing to be the required PR approver so the community can continue to contribute to it?

**Comment by trullock** [comment]
This is merged into my fork at https://github.com/trullock/chartjs-plugin-zoom which is published on npm as @trullock/chartjs-plugin-zoom, as this fork seems abandoned

**Comment by ikkala** [comment]
> This is merged into my fork at https://github.com/trullock/chartjs-plugin-zoom which is published on npm as @trullock/chartjs-plugin-zoom, as this fork seems abandoned

@trullock Cool!  And good that there's alternative fork in life now.

File: src/handlers.js [hunk 1]
```diff
@@ -58,6 +58,12 @@ function zoomStart(chart, event, zoomOptions) {
 }
 
 export function mouseDown(chart, event) {
+  const rect = event.target.getBoundingClientRect();
+  const offsetX = event.clientX - rect.left;
+  const offsetY = event.clientY - rect.top;
+  if (chart.legend && _isPointInArea({x: offsetX, y: offsetY}, chart.legend)) {
+    return;
+  }
```
Comment (kurkle) [inline src/handlers.js:66]:
```suggestion
  if (chart.legend) {
    const point = getRelativePosition(event, chart);
    if (_isPointInArea({x: offsetX, y: offsetY}, chart.legend)) {
    return;
    }
  } 
```

Comment (ikkala) [reply src/handlers.js:66]: Where does the `offsetX` and `offsetY` come from..?  And where do the `point` go?

Comment (kurkle) [reply src/handlers.js:66]: Sorry, my suggestion was incomplete. Fixed it.

Comment (ikkala) [reply src/handlers.js]: Yep, I tested the new one -- look good now, thanks!

**Review by kurkle — DISMISSED** [review]

**Review by kurkle — APPROVED** [review]

**Pull request merged**